haxepunk/graphics/hardware/HardwareRenderer.hx

```diff
@@ -70,9 +70,9 @@ class HardwareRenderer
 
 	public function new()
 	{
-		#if ios
-		defaultFramebuffer = new GLFramebuffer(GL.version, GL.getParameter(GL.FRAMEBUFFER_BINDING));
-		#end
+		//#if ios
+		//defaultFramebuffer = GL.createFramebuffer();
+		//#end
```

Comment:
Instead of commenting out, why not just delete these lines if they're not needed anymore?